src/js/libcs/General.js

```diff
@@ -266,8 +266,8 @@ General.identity = function (x) {
 
 General.deeplyEqual = function (a, b) {
     if (typeof a !== "object" || typeof b !== "object" || a === null || b === null) return a === b;
-    var cnt = 0;
-    var k;
+    let cnt = 0;
+    let k;
     for (k in a) {
```

Comment:
Change  `for (k in a)` to  `for (const k in a)` here and `for (k in b)` to  `for (const k in b)` below and remove `let k;` above. `k` is only used as a handle and is otherwise not changed.